tests/phpunit/tests/image/editorImagick.php

```diff
@@ -641,4 +641,43 @@ public function test_directory_creation() {
 
 		$this->assertNotWPError( $saved );
 	}
+
+	/**
+	 * @ticket 39216
+	 * Only affects systems with Ghostscript version >= 9.14.
+	 */
+	public function test_remove_alpha_pdf_preview() {
```

Comment:
```suggestion
	public function test_remove_pdf_alpha_channel_should_remove_the_alpha_chanel_in_preview() {
```
Just for a little more specificity should a failure occur.